### PR TITLE
[16.0][FIX] quality_control_oca: In product category form view, fix quality…

### PR DESCRIPTION
**Pull request by alfredoavanzosc**
… control tree.

### CODE REVIEW

File: quality_control_oca/views/product_category_view.xml [hunk 1]
```diff
@@ -12,21 +12,24 @@
...
-                            </tree>
-                        </field>
-                    </group>
-                </t>
+                <group col="2" />
+                <group
+                    name="qc"
+                    string="Quality control"
+                    groups="quality_control_oca.group_quality_control_user"
+                >
```
Comment (yostashiro) [inline quality_control_oca/views/product_category_view.xml]: I think we can omit the empty group.
```suggestion
                <group
                    name="qc"
                    string="Quality control"
                    groups="quality_control_oca.group_quality_control_user"
                    col="2"
                >
```

Comment (alfredoavanzosc) [reply quality_control_oca/views/product_category_view.xml]: Thanks @yostashiro

**Review by yostashiro — APPROVED** [review]
LGTM. 👍

**Review by anajuaristi — APPROVED** [review]

**Pull request merged**